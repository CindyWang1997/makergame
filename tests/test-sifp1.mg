int add(int a, int b)
{
  return a + b;
}

object main {
event create
{
  std::print_float(3+3.5);
  std::print_float(3.5+3);
  std::print_float(3+3);
  int b = 4.5;
  float c = b;
  std::print_float(c);
  std::print(add(39, 3.5));
  if (3.0 == 3) {
    std::printstr("3.0 == 3\n");
  }
  else {
    std::printstr("3.0 != 3\n");
  }
  std::end_game();
}
}
