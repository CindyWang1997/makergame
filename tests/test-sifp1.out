6.500000
6.500000
6.000000
4.000000
42
3.0 == 3

